packages/template-retail-react-app/app/pages/product-list/partials/checkbox-refinements.jsx

```diff
@@ -15,17 +15,14 @@ const CheckboxRefinements = ({filter, toggleFilter, selectedFilters}) => {
             {filter.values
                 ?.filter((refinementValue) => refinementValue.hitCount > 0)
                 .map((value) => {
+                    const isChecked = selectedFilters.includes(value.value)
+
                     return (
                         <Box key={value.value}>
                             <Checkbox
-                                isChecked={!!selectedFilters}
+                                isChecked={isChecked}
                                 onChange={() =>
-                                    toggleFilter(
-                                        value,
-                                        filter.attributeId,
-                                        !!selectedFilters,
-                                        false
-                                    )
+                                    toggleFilter(value, filter.attributeId, isChecked, false)
```

Comment:
I didn't apply it yet, but whilst making these changes on the project I'm working on I've also removed the final `false` argument (`allowMultiple`) from the `toggleFilter` method. Thus making it take the default of `true`.

This being because IMO checkboxes should represent an option that can have multiple items selected (and be deselected). Whereas currently they're essentially serving as radio buttons.

If you agree I can remove it here too.

Reply:
I'm open to this based on your reasoning. But that final argument is being used used when called from the radio-refinements component. Is your change away of this and retain the correct functionality in that component?

Reply:
Hey @bendvc, yeah it is, sorry my comment here wasn't very clear.

I don't mean removing the argument from the actual `toggleFilter` method, just removing the `false` argument here where it's called within `CheckboxRefinements` (so that the original method's default value is applied).

Reply:
@breadadams  Ok 💯 makes sense. I'm ok with that change. Go ahead and commit that, and we can get this merged.

Reply:
@bendvc, great! Resolved in latest commit 🙂